hathor/transaction/storage/transaction_storage.py

```diff
@@ -753,15 +1067,23 @@ def _topological_sort_dfs(self, root: BaseTransaction, visited: Dict[bytes, int]
             # matter.
             for parent_hash in tx.parents[::-1]:
                 if parent_hash not in visited:
-                    parent = self.get_transaction(parent_hash)
-                    stack.append(parent)
+                    try:
+                        parent = self.get_transaction(parent_hash)
+                    except TransactionDoesNotExist:
+                        pass
```

Comment:
Maybe add a comment explaining that it might fail because some dependencies might not exist.

Reply:
Done.